### DISCUSSION TITLE
Wireframing

### DISCUSSION THREAD

**Discussion by GeoffreyXue** [DSSD-Madison/Nagoya#12] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GeoffreyXue** [comment]
@cmg777 This is a test

**Comment by cmg777** [comment]
It works!
